### ISSUE TITLE
JAWS 2018 reads "x" when using MathJax in <label> with radio button

### ISSUE THREAD

**Issue by lud2k** [mathjax/MathJax#1874]
JAWS 2018 on IE11 does not read MathJax content that is in a \<label\> tag next to a radio button.
Instead it reads "x", "x radio button".

I have tried to find a solution to this issue but couldn't. Having "Supportive MathML" enabled does not change anything. Still reads "x radio button".

This can be reproduce with the following code with version 2.7.2 of MathJax.
```<!DOCTYPE html>
<html>
    <head>
        <title>MathJax TeX Test Page</title>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <script type="text/x-mathjax-config">
            MathJax.Hub.Config({
                tex2jax: {inlineMath: [["$","$"],["\\(","\\)"]]}
            });
        </script>
        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_HTML-full"></script>
    </head>
    <body>
        <form>
            <p>
                What is the correct answer?
            </p>
            <label>
                <input type="radio" name="answer" value="0" />
                <div>
                    \( 0.48 \)
                </div>
            </label>
            <label>
                <input type="radio" name="answer" value="1" />
                <div>
                    \( 1.16 \)
                </div>
            </label>
            <label>
                <input type="radio" name="answer" value="2" />
                <div>
                    \( 12.92 \)
                </div>
            </label>
            <label>
                <input type="radio" name="answer" value="3" />
                <div>
                    \( 13.53 \)
                </div>
            </label>
        </form>
    </body>
</html>
```

Any ideas what could be causing this?

Thanks,
Ludo.

**Comment by pkra** [comment]
JAWS hacks the DOM (in IE) to detect MathJax and then override various things to force its own math tool upon users; this causes many issues all of which are effectively impossible to debug. We generally don't have the resources to go down that particular rabbit hole too often. 

You might be able to find better advice (or file a bug) at https://github.com/FreedomScientific/VFO-standards-support/. Please try that first but let us know if that does not work out.

FYI `div` inside `label` is not valid HTML (that shouldn't really cause the issue but who knows, it's IE after all).

**Comment by lud2k** [comment]
Thanks for the quick answer. I have opened an issue on their side too.

**Comment by pkra** [comment]
Thanks. Sorry for not having a better answer.
